dapp/tests/auctions/auctions.spec.ts

```diff
@@ -1,11 +1,70 @@
 // Copyright (c) 2025 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-import { expect } from '@playwright/test';
+import { expect, test } from '../helpers/fixtures';
+import {
+    connectWallet,
+    createWallet,
+    generateRandomName,
+    requestFaucetTokens,
+    transactionToCreateAnAuction,
+} from '../utils';
 
-import { test } from '../helpers/fixtures';
+test.describe.serial('Auction Bid Flow', () => {
+    test.beforeAll(async ({ appPage, context, extensionPage, extensionName, sharedState }) => {
+        const { address, mnemonic } = await createWallet(extensionPage);
 
-test('Auction flow works', async ({ appPage }) => {
-    await expect(appPage).toHaveTitle(/IOTA Names/);
-    await expect(appPage.getByText('Your On-Chain Name')).toBeVisible();
+        await appPage.bringToFront();
+        await connectWallet(appPage, context, extensionName);
+
+        await requestFaucetTokens(address);
+        sharedState.wallet.address = address;
+        sharedState.wallet.mnemonic = mnemonic;
+
+        sharedState.testAuctionName = generateRandomName();
```

Comment:
sharedState is something you might want to use between different tests. this should not be one of these
```suggestion
```